src/lib/bootloader/grub2base.rb

```diff
@@ -120,6 +120,9 @@ def propose
 
       propose_serial
 
+      # we proposing new config and we do not know sections yet, so propose empty
+      @sections.default = ""
```

Comment:
well, sections by default after construction do lazy loading of settings. It is question of default if we should read it in constructor? but then we have to have some read method, which now do not exist.